digital-deck-server/services/session.js

```diff
@@ -0,0 +1,28 @@
+var cardDistService = require("./cardDistService.js");
+var Player = require("./player.js");
+
+/**
+ * Class that creates and manages a game session
+ */
+module.exports = class Session {
+  // Keeps track of distinct game sessions
+  static id = 0;
+  constructor(decks, players, cardsPerPlayer, cardsOnTable) {
+    this.numDecks = decks;
+    this.numPlayers = players;
```

Comment:
You should probably have a `maxNumPlayers` and `currentNumPlayers`, and have the `cardDistService` operate on `currentNumPlayers`

Reply:
I think we should just go opposite for now and just remove this parameter because this sprint we are just focusing on single player. We can deal with multi player logic next sprint as discussed.

Reply:
No I think you should keep the logic as is and just default the parameter to 1 for now. Else we'd have to add it back in later anyway and that's just unnecessary.